src/adhocracy_euth/adhocracy_euth/resources/idea_collection.py

```diff
@@ -15,6 +15,18 @@ class IProcess(process.IProcess):
 )
 
 
+class IPrivateProcess(process.IProcess):
+    """Private idea collection participation process."""
+
```

Comment:
.IPrivateProcess should subclass .IProcess, it is actually the same process type exept different permisions.